examples/build.gradle.kts

```diff
@@ -0,0 +1,73 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+group = rootProject.group
+version = rootProject.version
+
+fun properties(key: String) = project.findProperty(key).toString()
+
+plugins {
+    java
+    kotlin("jvm") version "1.5.21" apply true
+    id("org.jetbrains.intellij") version "1.1.3" apply true
+    id("org.jetbrains.dokka") version "1.4.30" apply true
+    id("org.jlleitschuh.gradle.ktlint") version "10.0.0" apply true
+}
+
+allprojects {
+    apply {
+        plugin("java")
+        plugin("kotlin")
+        plugin("org.jetbrains.intellij")
+        plugin("org.jetbrains.dokka")
+        plugin("org.jlleitschuh.gradle.ktlint")
+    }
+
+    repositories {
+        mavenCentral()
+        jcenter()
+    }
+
+    val utilitiesProjectName = "org.jetbrains.research.pluginUtilities"
+    dependencies {
+        implementation(kotlin("stdlib-jdk8"))
+
+        // Plugin utilities modules
+        implementation("$utilitiesProjectName:plugin-utilities-core") {
+            version {
+                branch = "add-python-sdk-util"
```

Comment:
But what branch should be specified if we want to test our library? `$CURRENT_BRANCH`? 0_0

Reply:
I am working on this now in another branch, please, wait for a little...

Reply:
@Furetur see it)